Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -168,6 +181,10 @@ std::optional<FloatSize> getVideoResolutionFromCaps(const GstCaps* caps)
         height = GST_VIDEO_INFO_HEIGHT(&info);
         pixelAspectRatioNumerator = GST_VIDEO_INFO_PAR_N(&info);
         pixelAspectRatioDenominator = GST_VIDEO_INFO_PAR_D(&info);
+    } else {
+        gst_structure_get_int(structure, "width", &width);
+        gst_structure_get_int(structure, "height", &height);
+        gst_structure_get_fraction(structure, "pixel-aspect-ratio", &pixelAspectRatioNumerator, &pixelAspectRatioDenominator);
```

Comment:
Return values should be checked.

Reply:
They don't seem to be getting checked elsewhere, not sure if that's actually needed here?:
https://github.com/WebKit/WebKit/blob/16de8a90c10880777b7e106e3e11b313cca02951/Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp#L158-L160